src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -85,32 +85,30 @@ public String getConfirmation(@PathVariable String companyNumber,
             return ERROR_VIEW;
         }
 
-        // If the payment is anything but paid return user to beginning of journey
-        PayableLateFilingPenalty payablePenalty;
-        CompanyProfileApi companyProfileApi;
         try {
-            companyProfileApi = companyService.getCompanyProfile(companyNumber);
-            payablePenalty = payablePenaltyService
-                    .getPayableLateFilingPenalty(companyNumber, penaltyId);
-        } catch (ServiceException ex) {
-            LOGGER.errorRequest(request, ex.getMessage(), ex);
-            return ERROR_VIEW;
-        }
+            PayableLateFilingPenalty payablePenalty = payablePenaltyService
+                    .getPayableLateFilingPenalty(companyNumber, payableRef);
 
-        if (!paymentStatus.equals("paid")) {
-            LOGGER.info("Payment status is " + paymentStatus + " and not of status 'paid', returning to beginning of journey");
-            return UrlBasedViewResolver.REDIRECT_URL_PREFIX + payablePenalty.getLinks().get("resume_journey_uri");
+            // If the payment is anything but paid return user to beginning of journey
+            if (!paymentStatus.equals("paid")) {
+                LOGGER.info("Payment status is " + paymentStatus + " and not of status 'paid', returning to beginning of journey");
+                return UrlBasedViewResolver.REDIRECT_URL_PREFIX + payablePenalty.getLinks().get("resume_journey_uri");
+            }
 
-        }
+            CompanyProfileApi companyProfileApi = companyService.getCompanyProfile(companyNumber);
 
-        model.addAttribute(COMPANY_NUMBER_ATTR, companyNumber);
-        model.addAttribute(PENALTY_NUMBER_ATTR, penaltyId);
-        model.addAttribute(COMPANY_NAME_ATTR, companyProfileApi.getCompanyName());
-        model.addAttribute(REASON_ATTR, PENALTY_REASON);
-        model.addAttribute(PAYMENT_DATE_ATTR, setUpPaymentDateDisplay(payablePenalty));
-        model.addAttribute(PENALTY_AMOUNT_ATTR, setUpPaymentAmountDisplay(payablePenalty));
+            model.addAttribute(COMPANY_NUMBER_ATTR, companyNumber);
+            model.addAttribute(PENALTY_NUMBER_ATTR, penaltyRef);
```

Comment:
Please can this model attribute be changed to match the new path param?
`PENALTY_REF_ATTR = "penaltyRef";`